src/Draco.Compiler/Internal/Symbols/IPropertyAccessorSymbol.cs

```diff
@@ -0,0 +1,12 @@
+namespace Draco.Compiler.Internal.Symbols;
+
+/// <summary>
+/// Represents an property accessor symbol, is intended to use while also inheriting <see cref="FunctionSymbol"/>.
```

Comment:
Ah ok I see what it is now, makes sense. Please add something like:
"For example, get, set and init are accessors in C#."